test/parallel/test-repl-eval.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const repl = require('repl');
+
+{
```

Comment:
Due to block-scoped vars (i.e. `let` and `const`) this feature, though existing in es5, is now actually useful.